app/coffee/Features/Messages/MessageManager.coffee

```diff
@@ -27,10 +27,41 @@ module.exports = MessageManager =
 			room_id: { $in: room_ids }
 		}, callback
 
+	deleteAllMessagesInRoom: (room_id, callback = (error) ->) ->
+		db.messages.remove {
+			room_id: room_id
+		}, callback
+	
+	updateMessage: (room_id, message_id, content, timestamp, callback = (error, message) ->) ->
+		query = @_ensureIdsAreObjectIds(
+			_id: message_id
+			room_id: room_id
+		)
+		db.messages.update query, {
+			$set:
+				content: content
+				edited_at: timestamp
+		}, (error) ->
+			return callback(error) if error?
+			db.messages.find query, (error, messages) ->
+				return callback(error) if error?
+				return callback null, messages[0]
```

Comment:
Actually, we don't use the return value, it's left over from an earlier implementation. Removed this code entirely.